packages/docs/src/styles/_examples.scss

```diff
@@ -193,8 +193,12 @@ $options-margin: $pt-grid-size * 3;
 }
 
 .docs-tabs2-example {
-  .pt-navbar .pt-tab {
-    line-height: $pt-grid-size * 5;
+  .pt-navbar {
+    position: static;
```

Comment:
move this more generally in docs examples styles to hit http://blueprintjs.com/docs/#components.navbar too